### ISSUE TITLE
Cannot read parameter "default_crop_class" for module maize Fatal Error

### ISSUE THREAD

**Issue by smkuhlani** [APSIMInitiative/APSIMClassic#2236]
Hallo ApsimERS,

I am running a simulation in APSIM 7.10 but i am getting an error below: 

                  APSIM  Fatal  Error
                      -------------------
     Cannot read parameter "default_crop_class" for module maize.
     Component name: paddock.maize
I have attached the apsim giving me such an error.

[maize_current.zip](https://github.com/APSIMInitiative/APSIM710/files/15212898/maize_current.zip)

**Comment by peter-devoil** [comment]
That looks like an older version of Maize.xml - certainly it's not like the [current one](https://github.com/APSIMInitiative/APSIM710/blob/master/Model/Maize.xml). Perhaps it was from the cropmod days (ie apsim pre 7.9)?

**Comment by smkuhlani** [comment]
Yes, this was 7.9. Is there a way to convert 7.9 files in 7.10? 

[Maize_Sbean_current.zip](https://github.com/APSIMInitiative/APSIM710/files/15236301/Maize_Sbean_current.zip)

**Comment by peter-devoil** [comment]
There's no automated way. The procedure is to identify what changes have been made to the 7.9 file and hope that similar changes can be made to the new maize model, eg add a new cultivar. If it's just phenological targets, then that's not too difficult, but leaf area & N conc are totally different..
